workflow/controller/operator.go

```diff
@@ -1469,6 +1475,13 @@ func (woc *wfOperationCtx) executeTemplate(nodeName string, orgTmpl wfv1.Templat
 
 	node := woc.wf.GetNodeByName(nodeName)
 
+	woc.currentStackDepth++
```

Comment:
nice

Reply:
Even if there is, I don't think we should use it since we're concerned with our own definition of "stack" -- i.e., a call to `executeTempalte`. If we use something like `runtime.GetStacks()` we could get red-herrings and have an inaccurate stack counter.